pages/congrats/index.tsx

```diff
@@ -31,13 +31,14 @@ function ConfirmationPage(): JSX.Element {
       (window as any).dataLayer = (window as any).dataLayer || [];
       (window as any).dataLayer.push({
         event: "library_card_submission",
-        nypl_location: libraryName[0]?.label,
+        nypl_location: libraryName[0]?.label || "E-branch default",
+        location_id: state.formValues.homeLibraryCode,
```

Comment:
Can you add a comment that the fallback value is in the `ilsLibraryList` array? It sticks out when the `nypl_location` has an explicit default value but `location_id` does not.

Reply:
Sure! I'm waiting on confirmation from Miguel what we want the default label to be.

Reply:
I'm going to merge these changes for now, since there is other work that needs QAing, but I'm still waiting on updated requirements from Miguel.